rules/windows/execution_unusual_dns_service_file_writes.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2020/07/16"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/16"
+
+[rule]
+author = ["Elastic"]
+description = """
+An unexpected file was modified by dns.exe, which is responsible for DNS Server services. This could indicate that the
+service has been maliciously modified or controlled.
+"""
+index = ["winlogbeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "Unusual File Modification by dns.exe"
+references = [
```

Comment:
```suggestion
note = """### Investigating Unusual File Write
Detection alerts from this rule indicate potential unusual/abnormal file writes from the DNS Server service process (dns.exe) after exploitation from CVE-2020-1350 (SigRed) has occurred. Here are some possible avenues of investigation:
- After exploitation, adversaries may write additional files or payloads to the system to drop associated files used for additional discovery/exploitation/persistence. 
- Any suspicious or abnormal file written from dns.exe should be reviewed and investigated with care."""
references = [
```

Reply:
I made a few changes to make it easier to read.